### PR TITLE
Simple grammar fixes

### PR DESCRIPTION
**Pull request by tyirenkyi**

### CODE REVIEW

**Review by aftr101 — APPROVED** [review]

File: Solana_NFTs/en/Section_0/Lesson_2_Whats_Different_About_Solana_NFTs.md [hunk 1]
```diff
@@ -31,10 +31,10 @@ One thing that's very special about the candy machine is it won't accept a user'
...
 6. Person A loses the race and is sad, and has now lost money on an NFT they never received. Person B is happy.
 
-This is classic problem is parallel computing. The fix is to use something called a [mutex](https://doc.rust-lang.org/std/sync/struct.Mutex.html) along with an [atomic transaction](https://en.wikipedia.org/wiki/Atomicity_(database_systems)) which are both decently complex to implement.
+This is a classic problem in parallel computing. The fix is to use something called a [mutex](https://doc.rust-lang.org/std/sync/struct.Mutex.html) along with an [atomic transaction](https://en.wikipedia.org/wiki/Atomicity_(database_systems)) which are both decently complex to implement.
 
 **Metaplex's candy machine implements this stuff for us :).**
 
-Candy machine also has some other goodies it gives us as well which we'll cover later but hope this overview helps later down the line!! Feel free to come back to it once you get more into the candy machine stuff.
+Candy machine also has some other goodies it gives us as well which we'll cover later but I hope this overview helps later down the line!! Feel free to come back to it once you get more into the candy machine stuff.
```
Comment (adilanchian) [inline Solana_NFTs/en/Section_0/Lesson_2_Whats_Different_About_Solana_NFTs.md:38]: can you add a comma after later here please?

**Review by adilanchian — CHANGES_REQUESTED** [review]
Nice catches here :)

Left a very small comment for ya if you don't mind!

File: Solana_NFTs/en/Section_0/Lesson_2_Whats_Different_About_Solana_NFTs.md (hunk 1 above)
Comment (tyirenkyi) [reply Solana_NFTs/en/Section_0/Lesson_2_Whats_Different_About_Solana_NFTs.md:38]: Sure, on it

Comment (tyirenkyi) [reply Solana_NFTs/en/Section_0/Lesson_2_Whats_Different_About_Solana_NFTs.md]: done

**Comment by adilanchian** [comment]
Cheers! Thanks for this :)

**Comment by adilanchian** [comment]
ALSO CONGRATS ON YOUR FIRST CONTRIBUTION

✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨

**Comment by adilanchian** [comment]
@all-contributors please add @tyirenkyi for doc

**Pull request merged**